### ISSUE TITLE
Support for new SmartThings device integration

### ISSUE THREAD

**Issue by brianfromoregon** [iharyadi/zigbee_repeater#1]
Hi Iman
I have one of your repeaters and wanted to ask if you had any plan to provide integration with SmartThings new developer APIs that will eventually replace the groovy device handlers?

https://smartthings.developer.samsung.com/docs/tools/sdks.html
https://www.thedigitalmediazone.com/2020/06/29/smartthings-to-retire-legacy-platforms/

**Comment by iharyadi** [comment]
Hi Brian,

As a developer,  I would love to support the new ST platform.  I do have to
find time to dedicate for this..  It also depends on how complex the new
platform is.

The link that you sent form ST is for hosting an app.  I am also interested
in porting my device driver to a new platform.  I am not sure how to do
this.  I am not clear as a device I have to host the driver and its
execution.  Or,  the user will download a code from me and host on their
own.

I would welcome help with any information specific to the device driver
migration.  In the past,  I started working on the DTH when the development
platform was already clear to developers like me.  Now,  I feel there is
some significant new stuff that I need to catch up.   I also have been
working on Hubitat extending my sensor capability in the scope of an
existing development platform which we all are already familiar with.

In summary,  I am sincerely interested in making my DTH compatible with the
new ST platform.  However,  I can't commit to do so at this moment until
the effort and cost to do so is clear to me.  At that point,  if the cost
is something affordable to me,  I will do it.

Thanks
Iman

**Comment by brianfromoregon** [comment]
Thanks for the reply. I think the docs are here, a quick glace makes me think you'll be able to reuse your groovy code.
https://smartthings.developer.samsung.com/docs/devices/hub/hub-connected-device.html

If you decide to give this a shot at some point, in case it helps I have an organization registered in ST and can add you to it, if that makes things easier for you.
